src/test/java/org/cactoos/io/HeadInputStreamTest.java

```diff
@@ -74,11 +80,13 @@ public void testResetting() throws Exception {
         );
         stream.skip(7L);
         stream.reset();
-        MatcherAssert.assertThat(
+        new Assertion<>(
             "Reset didn't change the state",
-            new TextOf(stream),
+            () -> new TextOf(
+                new Sticky(() -> stream)
```

Comment:
@Iprogrammerr Is necessary use the `Sticky` decorator here?

Reply:
@fabriciofx It is. Assertion calls text.asString() twice and second the call will fail, because stream will be closed.

Reply:
@Iprogrammerr Let's wait what @llorllale thinks about [cactoos-matchers#86](https://github.com/llorllale/cactoos-matchers/issues/86).